packages/rocketchat-lib/server/startup/settings.js

```diff
@@ -1503,6 +1503,18 @@ RocketChat.settings.addGroup('Push', function() {
 			}
 		]
 	});
+	this.add('Push_gateway_proxy', '', {
```

Comment:
I wonder if we shouldn't make this more generic.  The oembed fetch and a few others also would benefit from a proxy setting